### PR TITLE
Tmp/andre setup

### PR DESCRIPTION
**Pull request by andre-merzky**
Sync `setup.py` with RU's `setup.py`.

The only semantic change is that all examples are installed (incl. `misc/`), and the removal of the additional `scripts/` subdir in each example dir.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -173,90 +206,99 @@ def visit((prefix, strip, found), dirname, names):
...
-    'keywords'         : "ensemble workflow execution",
-    'classifiers'      :  [
+    'name'               : name,
+    'version'            : version,
+    'description'        : "Radical Ensemble Toolkit.",
+    'long_description'   : (read('README.md') + '\n\n' + read('CHANGES.md')),
+    'author'             : 'RADICAL Group at Rutgers University',
+    'author_email'       : 'radical@rutgers.edu',
+    'maintainer'         : 'The RADICAL Group',
+    'maintainer_email'   : 'radical@rutgers.edu',
```
Comment (iparask) [inline setup.py]: Does this email exist?

File: setup.py [hunk 2]
```diff
@@ -173,90 +206,99 @@ def visit((prefix, strip, found), dirname, names):
...
+    'packages'           : find_packages('src'),
+    'package_dir'        : {'': 'src'},
+    'scripts'            : ['bin/entk-version'],
+    'package_data'       : {'': ['*.sh', '*.json',
+                                 'VERSION', 'SDIST', sdist_name]},
+    'setup_requires'     : ['pytest-runner'],
+    'install_requires'   : ['radical.utils',
+                            'radical.pilot',
+                            'pika',
+                            'sphinx'
```
Comment (iparask) [inline setup.py]: I'm still not sure why `sphinx` is needed for installation. It is used for documentation generation.

File: src/radical/entk/__init__.py [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
-from radical.entk.version import version, __version__
+from .version import version, __version__, version_short, version_detail
+from .version import version_base, version_branch, sdist_name, sdist_path
 
-from radical.entk.pipeline.pipeline import Pipeline
-from radical.entk.stage.stage import Stage
-from radical.entk.task.task import Task
+from .pipeline.pipeline import Pipeline
+from .stage.stage       import Stage
+from .task.task         import Task
```
Comment (iparask) [inline src/radical/entk/__init__.py]: These imports are relative, but the one in line 9 is not. Shouldn't it be relative for consistency. Although, I think `pylint` and `pep8` complain with relative imports

File: src/radical/entk/pipeline/__init__.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (iparask) [inline src/radical/entk/pipeline/__init__.py]: Can a banner be placed here?

File: setup.py (hunk 1 above)
Comment (andre-merzky) [reply setup.py]: Good point - I thought so, but we can also use a different one, like one of our lists.  What I do know is that this specific email has a finite lifetime :-)

File: setup.py (hunk 2 above)
Comment (andre-merzky) [reply setup.py]: Ack.

File: src/radical/entk/__init__.py (hunk 3 above)
Comment (andre-merzky) [reply src/radical/entk/__init__.py]: Why do they complain?  
Yes, I will look at `states`.

File: setup.py (hunk 1 above)
Comment (vivek-bala) [reply setup.py]: Makes sense. Which one should we use? I would think maintainer email should be of an individual right or a group email (not rct-devel or something).

File: setup.py (hunk 2 above)
Comment (vivek-bala) [reply setup.py]: Ack. Will change.

File: src/radical/entk/pipeline/__init__.py (hunk 4 above)
Comment (vivek-bala) [reply src/radical/entk/pipeline/__init__.py]: What do you mean by a banner?

**Pull request closed without merging**